### ISSUE TITLE
What is the status of Nuget package XBim.Essentials.4?

### ISSUE THREAD

**Issue by Jero9999** [xBimTeam/XbimEssentials#53]
I notice that there is an XBim.Essentials.4 nuget package which is much newer than the standard XBim.Essentials package. However I don't see any formal release in the releases area of this project. 
It seems that most of the XBim repos have a .4 nuget package as well.
Can anyone expand on the status of these packages? Are they superseding the standard (not .4) packages?

Also, is there anywhere that details some of the API changes? For example I see that there are still transactions but there is no IfcStore.Validate() whereas there was a XBimModel.Validate(). Is validation just not there anymore?

Thanks
Jero

**Comment by Jero9999** [comment]
Hey thanks for the great reply Martin. 
My reference to the release area was in the github web UI it has releases at the top 
![image](https://cloud.githubusercontent.com/assets/4205884/19288332/09fffdd2-9062-11e6-85eb-f50a9787dac8.png)


and the latest listed release there is from back in 2015. That's where sonme of my confusion came from.

**Comment by CBenghi** [comment]
Hello All,
just to clarify here that the choice to have .4 versions of the nuget packages has been reverted, and we have resumed using the old Xbim.<PackageName> ids.
If you have upgraded to .4 the way to process is to 

1.  upgrade all libraries to the latest version.
2.  close and reopen the solution
3.  remove all the .4 packages without removing dependencies and with the -force uninstall flag set.

You'll find instructions on how to do this in the nuget.4 descriptions as well.
